components/recents/WindowHeight.tsx

```diff
@@ -0,0 +1,22 @@
+import { useState, useEffect } from "react";
+
+function getWindowDimensions() {
+  const height = innerHeight 
+  return height;
+}
+
+
+export default function useWindowDimensions() {
+  const [windowDimensions, setWindowDimensions] = useState(getWindowDimensions());
+
+  useEffect(() => {
```

Comment:
useEffect is for React components. If you're not using useWindowDImensions as a React component, then you don't need to wrap anything in a useEffect.

The effect is that windowDimensions is returned and the useEffect hook never even runs in the first place